superset/models/reports.py

```diff
@@ -102,6 +102,7 @@ class ReportSchedule(Model, AuditMixinNullable):
     context_markdown = Column(Text)
     active = Column(Boolean, default=True, index=True)
     crontab = Column(String(1000), nullable=False)
+    report_format = Column(String(50), default=ReportDataFormat.VISUALIZATION)
```

Comment:
Have you changed the format to be applied to all types of notifications, instead of letting the user choose the format to be sent by email and a possible different one for Slack for example?

Reply:
the format will apply to all types of notifications